azure-pipelines.yml

```
@@ -167,6 +167,9 @@ jobs:
       brew install flex bison mpich
       brew unlink mpich && brew install openmpi
       cmake --version
+      # see https://github.com/BlueBrain/CoreNeuron/issues/817, uninstall libomp until we fix this
+      # as we are building wheels, we shouldn't enable OpenMP here anyway
+      brew uninstall libomp || echo "libomp doesn't exist"
```

As @pramodk  was mentioning, I think we should go with:
```suggestion
      brew uninstall --ignore-dependencies libomp || echo "libomp doesn't exist"
```

---

Also in the logs
```
2022-05-12T18:47:40.9172480Z CMake suite maintained and supported by Kitware (kitware.com/cmake).
2022-05-12T18:47:43.6165900Z Error: Refusing to uninstall /usr/local/Cellar/libomp/14.0.0
2022-05-12T18:47:43.6166760Z because it is required by imagemagick and libraw, which are currently installed.
2022-05-12T18:47:43.6167520Z You can override this and force removal with:
2022-05-12T18:47:43.6169320Z   brew uninstall --ignore-dependencies libomp
2022-05-12T18:47:43.6826740Z libomp doesn't exist
```